File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        highest_count = 0
        num_hash = set(nums)

        for n in nums:
            if n - 1 not in num_hash and n in num_hash:
                count = 0
                while n in num_hash:
                    count += 1
                    n += 1

                if count > highest_count:
                    highest_count = count
        return highest_count
            
                


        

File: Data Structures & Algorithms/number-of-one-bits/submission-0.cs
public class Solution {
    public int HammingWeight(uint n) 
    {
        var count = 0;

        while (n > 0)
        {
            var currAnd = n & 1;
            if ( currAnd == 1 )
                count++;
            n = n >> 1;
        }

        return count;
    }
}
